src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -26,8 +27,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     LanguageVersion.VisualBasic11,
                     LanguageVersion.VisualBasic12,
                     LanguageVersion.VisualBasic14,
-                    LanguageVersion.VisualBasic15
-
+                    LanguageVersion.VisualBasic15,
+                    LanguageVersion.VBvnext
```

Comment:
@gafter
This is a **Temporary** additional language version, to "fix" an issue in the checking of non `LanguageVersion`-ed `/feature` flags. If it that PR is approved this can be removed, and this PR amended, or a patch PR implemented.